ci/release/_install.sh

```diff
@@ -149,49 +155,75 @@ main() {
   PREFIX=${PREFIX:-/usr/local}
   CACHE_DIR=$(cache_dir)
   mkdir -p "$CACHE_DIR"
+  METHOD=${METHOD:-detect}
   INSTALL_DIR=$PREFIX/lib/d2
 
+  case $METHOD in
+    detect)
+      case "$OS" in
+        macos)
+          log "detected macOS, using homebrew for (un)installation"
+          METHOD=homebrew
+          ;;
+        *)
+          warn "unrecognized OS $OS, falling back to --method=standalone"
+          METHOD=standalone
+          ;;
+      esac
+      ;;
+    standalone) ;;
+    homebrew) ;;
+    *)
+      echoerr "unknown (un)installation method $METHOD"
+      return 1
+      ;;
+  esac
+
   if [ -n "${UNINSTALL-}" ]; then
     uninstall
-    return 0
-  fi
-
-  VERSION=${VERSION:-latest}
-  if [ "$VERSION" = latest ]; then
-    header "fetching latest release info"
-    fetch_release_info
+  else
+    install
   fi
-
-  install
 }
 
 install() {
-  install_d2
-  if [ -n "${TALA-}" ]; then
-    # Run in subshell to avoid overwriting VERSION.
-    TALA_VERSION="$( install_tala && echo "$VERSION" )"
-  fi
+  case $METHOD in
+    standalone)
+      install_d2_standalone
+      if [ -n "${TALA-}" ]; then
+        # Run in subshell to avoid overwriting VERSION.
+        TALA_VERSION="$( RELEASE_INFO= install_tala_standalone && echo "$VERSION" )"
+      fi
+      ;;
+    homebrew)
+      install_d2_brew
+      if [ -n "${TALA-}" ]; then install_tala_brew; fi
+      ;;
+  esac
+
+  FGCOLOR=2 bigheader 'next steps'
```

Comment:
![Screen Shot 2022-11-20 at 7 03 46 PM](https://user-images.githubusercontent.com/3120367/202954463-e136c44f-1a75-49a3-b383-14559f68bd13.png)

we should only show this when there are next steps, e.g. d2 not found in path.

Reply:
What part are you referring to? The communication links?

Reply:
the "next steps". like there is no next steps. they're done, it's a complete success. the logs after are just "extra"

Reply:
I disagree, next steps is useful as a summary/intro.

We can debate further later, for now I'll merge and get it in use so we can see how people feel.